README.md

```diff
@@ -0,0 +1,187 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 CAPSTONE PROYECT <a name="about-project"></a>
+
+**[CAPSTONE PROYECT Urban Art Festival Creativity in the Streets 2023]** is a proyect was created to exercise good code and version control practices in the area of ​​web development.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>VSC</li>
+    <li>Node.js</li>
+    <li>Flexblox</li>
+    <li>Figma</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+-**Learn to use new tools for the frontend**
+-**Learn to make a professional webpage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://fernando426.github.io/Portfolio-mobile-website2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ - Git
+ - GitHub
+ - Any code editor
+ - Intalled Node.js
+
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/Fernando426/Portfolio-mobile-website.git
+
+Example commands:
+  cd my-folder
+  https://github.com/Fernando426/Portfolio-mobile-website.git
+
+### Install
+
+*Install this project with:
+- Any code editor
+- Node.js
+- Git
+- GitHub
+
+### Usage
+
+To run the project, execute the following command:
+
+you just have to double click on index.html file
+
+### Run tests
+
+To run tests, run the following command:
+
+you can run and verify the correct functioning of the linters from the command console using:
+npx stylelint "**/*.{css,scss}"
+
+to run the project you just have to double click on the file index.html
+
+### Deployment
+
+You can deploy this project using:
+
+-Git
+-Github
+-The text editor of your choice
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+👤 **Fernando Montes**
+
+- GitHub: [@Fernando426](https://github.com/Fernando426)
+- Twitter: [@Fernando_426_](https://twitter.com/Fernando_426_)
+- LinkedIn: [Fernando Montes](https://www.linkedin.com/in/fernando-jose-montes-linares-523822270/?originalSubdomain=ve)
+
+👤 **Jhonatan Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
```

Comment:
- [ ] please, like the previous reviewer noted, kindly add the author title and remove one author as this is an individual project.